### PR TITLE
feat: update the react native tutorial for v8

### PR DESCRIPTION
**Pull request by dannyhw**
This pr updates the en react native tutorial to v8, since the api is the same there aren't that many changes required and most the changes are updating images to match the redesigned ui

these changes go together with this pr here:
https://github.com/chromaui/intro-storybook-react-native-template/pull/5

### CODE REVIEW

**Comment by jonniebigodes** [comment]
@dannyhw thanks for taking the time to put together this pull request and update the tutorial's version. I was about to message you as I saw the updates to the tutorial, but you beat me to it 🙂. If you're ok with it, I'll take a look at it tomorrow morning my time, and if nothing stands out we can merge both at the same time and get everything published. Sounds good to you?

Looking forward to hearing from you.

Have a great day.

Stay safe

**Comment by dannyhw** [comment]
Hey, tomorrow is great :). However let's hold off on releasing it as im still working on the release post for v8 and need to get an idea of the content schedule.

Thanks for jumping on it so quick.

File: content/intro-to-storybook/react-native/en/simple-component.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ Now that we've set up the basics lets re-run `yarn storybook` and see our change
 You should see a UI that looks like this:
 ![a gif showing the task component in storybook](/intro-to-storybook/react-native-task-component.gif)
 
-You can use the Sidebar tab to swap between stories, the Canvas tab to see the current story and the addons tab to interact with args and actions.
+You can use the menu at the bottom of your screen to open the navigation menu and tap to swap between stories, then you can tap away or drag down to dismiss the menu. You'll find addons by pressing the icon on the far right of the bottom bar.
```
Comment (jonniebigodes) [inline content/intro-to-storybook/react-native/en/simple-component.md:124]: @dannyhw small item here if you don't mind.
```suggestion
You can use the menu at the bottom of your screen to open the navigation menu and tap to swap between stories. Then, you can tap away or drag down to dismiss the menu. You'll find addons by pressing the icon on the far right of the bottom bar.
```

**Review by jonniebigodes — APPROVED** [review]
@dannyhw I just left a small item for you to look into when you can. Nothing significant. Let me know once you've addressed it so we can coordinate to publish the updated version.


Have a great day.

Stay safe

File: content/intro-to-storybook/react-native/en/simple-component.md (hunk 1 above)
Comment (dannyhw) [reply content/intro-to-storybook/react-native/en/simple-component.md]: adjusted it, thanks!

**Comment by dannyhw** [comment]
Hey @jonniebigodes, I've released the new version so I think we're good to merge this.

**Comment by jonniebigodes** [comment]
Sure thing @dannyhw , I'll get this one merged in. I saw the notification drop-in. Thanks once again for taking the time to update the tutorial. Appreciate it 🙏.

Have a great weekend.

Stay safe

**Comment by dannyhw** [comment]
Thanks :)

**Pull request merged**